pkgs/games/gnoll/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, python39, flex, bison }:
+
+stdenv.mkDerivation rec {
+  pname = "gnoll";
+  version = "4.3.4";
+
+  src = fetchFromGitHub {
+    owner = "ianfhunter";
+    repo = "GNOLL";
+    rev = "v${version}";
+    sha256 = "1s495qmakw2n50x11klf6wb2869b5v94wsihq4r19mqdbg1pxm1y";
+  };
+
+  buildInputs = [ python39 flex bison ];
```

Comment:
I think that it's slightly more appropriate to have these as `nativeBuildInputs`, as they are required for compilation and not at runtime (right? I tested it and gnoll seems to work, but I'm not familiar with it).